observability/build.gradle

```diff
@@ -8,12 +8,31 @@ plugins {
 
 description = "gRPC: Observability"
 dependencies {
-    api project(':grpc-api')
+    def cloudLoggingVersion = '3.6.1'
 
-    implementation libraries.guava,
-            project(':grpc-protobuf'),
+    api (project(':grpc-api')) {
+        exclude group: 'com.google.guava', module: 'guava'
```

Comment:
DO we still need this exclude line and the ones on 18 and 19? With the explicit version dependencies we added (on lines 22-29) we should not need these excludes. Pls verify and update accordingly.